### DISCUSSION TITLE
VS Code giving warnings when using nested styles

### DISCUSSION THREAD

**Discussion by sharathdaniel** [tailwindlabs/tailwindcss#11868] [Help]
Hi,

I've setup tailwind in Angular with **scss**, but when writing nested styles, i'm getting these warnings:
![image](https://github.com/tailwindlabs/tailwindcss/assets/5154064/097583f9-cfc5-49a4-887c-59ea20119679)

Below is the code:
![image](https://github.com/tailwindlabs/tailwindcss/assets/5154064/a87c9cc2-e262-42e3-80e8-b225a7269e61)

Here is my postcss.config.js
```javascript
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': {},
    tailwindcss: {},
    autoprefixer: {},
  }
}
```

This is how i'm importing the styles:
![image](https://github.com/tailwindlabs/tailwindcss/assets/5154064/5bd3dc7e-6153-489d-8696-272e0d7ef7e8)

This is my folder structure for scss files:
![image](https://github.com/tailwindlabs/tailwindcss/assets/5154064/1cb1c88f-f0c7-4fd5-90ef-f2f1898e31cd)
**main.scss** is the entry point for styles.

Am i doing this entirely wrong or does it need more setup? Even with the warnings the styles are getting applied somehow.

**Comment by wongjn** [comment]
If the styles are still getting applied then you're fine. The warnings are from VSCode's Sass syntax checker and it seems like it does not understand `@apply` syntax.

**Reply by sharathdaniel** [reply]
Even without `@apply` its showing these warnings.

**Reply by wongjn** [reply]
I took a closer look. It seems like you have configured Tailwind CSS Intellisense for this Sass file, but it does not understand the nesting syntax of Sass/Scss.

**Reply by sharathdaniel** [reply]
But intellisense is needed, so what are my options?

**Reply by wongjn** [reply]
You could consider not using Scss syntax.
